gatorgrouper/templates/gatorgrouper/home.html

```diff
@@ -59,16 +59,37 @@ <h1 class="ont-weight-light">Developer Story</h1>
 <section class="py-5">
     <div class="container">
       <h1 class="font-weight-light">Product Desrcription</h1>
+<<<<<<< HEAD

```

Comment:
This file is where the merge markers were introduced. You need to manually remove them and the duplicate content -- remember that the version above the `=====` is from your branch, while the things below are from master.

Reply:
Since I worked with @finneyj2 on this I went ahead and merged home.html

Reply:
@quigley-c Thank you for merging home.html. I'll test this later today, and update this if it's fixed